lib/file.js

```diff
@@ -102,11 +102,12 @@ File.prototype.dependencies = function () {
  */
 
 File.prototype.load = function *() {
-  if (this.src) return this;
+  if (typeof this.src !== 'undefined') return this;
```

Comment:
I was addressing the concern that just checking `undefined` excludes the possibility of `null`